subfunctions/demo_CaBBI_FHN

```diff
@@ -0,0 +1,236 @@
+inF.Tau_Ca0 = TauCa_realTime*dt/dt_real;                                   % effective prior mean of tau_Ca in inversion; see Eqn. 17
+inG.k_F0 = 5;                                                              % scale parameter of the fluorescence observatios 
+inG.ind  = 3;                                                              % index of [Ca2+] variable
+
+
+%% Step 6. Setting the internal parameters of the optimization (VB-Laplace) method
+options.backwardLag = 2;                                                   % lag parameter (selcet either 2 or 3); increasing it e.g. to 3 (or higher) "may" provide more accurate inversions 
+options.updateHP    = 1;                                                   % this allows the precison (hyper) parameters to be learnt from data
+options.MaxIterInit = 0;  
+options.MaxIter     = 2;  % or e.g. 3                                      % maximum number of optimization iterations; increasing it will lead to a  
+                                                                           % complete optimization process, although it may take long time to be converged)
+                                                                           
+% options.GnFigs      = 0;                                                 % set to zero to speed up the inversion                                                  
+options.DisplayWin  = 0;                                                   % set to zero to speed up the inversion
+options.verbose     = 1;                                                   % set to zero to speed up the inversion
+options.decim       = 1;                                                   % this determine the micro-time resolution; increasing it e.g. to 2 (or higher) "may" provide more accurate inversions
+
+
+%% Step 7. Assigning the prior densitities:  (mu --> mean), (Sigma --> variance)
+% prior densities for initial conditons
+priors.muX0         = [-1.2; -0.62; 0.05];                                 % [V(0), W(0), [Ca2+](0)]
+priors.SigmaX0(1,1) =  4;
+priors.SigmaX0(2,2) =  1;
+priors.SigmaX0(3,3) =  25;
+
+% prior densities for evolution parameters
+priors.muTheta         = [0; 0; 0];  
+priors.SigmaTheta(1,1) = 5;                                                % for X_Tau_Ca
+priors.SigmaTheta(2,2) = 100;                                              % for Ca_base 
+priors.SigmaTheta(3,3) = 5;                                                % for X_k_Ca 
+
+% prior densities for observation parameters
+priors.muPhi         = [min(Fluor_trace)/2;0];
+priors.SigmaPhi(1,1) = 0.25;                                               % for d_F
+priors.SigmaPhi(2,2) = 1;                                                  % for X_k_F
+
+% prior densities for precision parameters: Gamma distributions
+priors.a_alpha = 1e0;
+priors.b_alpha = 1e0;
+priors.a_sigma = 1e0;
+priors.b_sigma = 1e0;
```

Comment:
You're right. However, I think it would be better that we don't change these values, in order to stay consistent with the prior sets we provided in the paper.